src/target/ch579.c

```diff
@@ -136,7 +136,7 @@ bool ch579_probe(target_s *target)
 {
 	uint8_t chip_id = target_mem32_read8(target, CH579_R8_CHIP_ID);
 	if (chip_id != 0x79) {
-		DEBUG_ERROR("Not CH579! 0x%02" PRIx8 "\n", chip_id);
+		DEBUG_ERROR("Not CH579! 0x%02x\n", chip_id);
```

Comment:
After conversing with @dragonmux , We concluded that it is not necessary to use PRI*8 specifiers and better to stick to %d/%u/%x